rc/filetype/svelte.kak

```diff
@@ -0,0 +1,22 @@
+hook global WinCreate .*\.svelte %[
+    set-option window filetype svelte
+]
+
+hook global WinSetOption filetype=(svelte) %{
+    require-module html
+
+    hook window ModeChange pop:insert:.* -group "%val{hook_param_capture_1}-trim-indent"  html-trim-indent
```

Comment:
Any strong reason to use hook capture and string expansions here instead of just hard-coding "svelt" for those hook groups ?

Reply:
Not really, I kept it in so it could support different filetypes in the future, but I don't think there are any other filetypes for svelte. 
So I'll hardcode it.